util_test.go

```diff
@@ -391,6 +391,12 @@ func skipOnJenkins(t *testing.T, message string) {
 	}
 }
 
+func runOnlyOnDockerContainer(t *testing.T, message string) {
+	if os.Getenv("DOCKER_CONTAINER_SETUP") == "" {
```

Comment:
Shouldn't we add this env to `docker run` command? Maybe something more specific like `AUTHENTICATION_TESTS_ENV`?